karabo_bridge/cli/glimpse.py

```diff
@@ -175,13 +175,15 @@ def main(argv=None):
                     "structure. optionally: save its data to an HDF5 file.")
     ap.add_argument('endpoint',
                     help="ZMQ address to connect to, e.g. 'tcp://localhost:4545'")
+    ap.add_argument('--sock', default='REQ', choices=['REQ', 'SUB', 'PULL'],
+                    help='Socket type (default REQ)')
```

Comment:
I wonder if it would be clearer to specify the socket type that the server is using (REP/PUB/PUSH) rather than the socket type the client should use. If people don't care about the implementation details, it's easier to know 'this needs to match a setting on the server' than to think about matching up names.

It's maybe clearer for someone who knows ZMQ to specify the client socket type, but even so I don't think it would be a problem so long as it's clearly documented in the help string that you're describing the server socket.

Reply:
That's a good point, thanks. I'll follow that.